browser-test/src/applicant/questions/id.test.ts

```diff
@@ -184,4 +177,60 @@ test.describe('Id question for applicant flow', () => {
       await validateAccessibility(page)
     })
   })
+
+  test.describe('single id question with north star flag enabled', () => {
+    const programName = 'Test program for single id'
+
+    test.beforeAll(async () => {
+      await setUpForSingleInput(programName)
+    })
+
+    test.beforeEach(async () => {
+      const {page} = ctx
+      await enableFeatureFlag(page, 'north_star_applicant_ui')
+    })
+
+    test(
+      'validate screenshot with north star flag enabled',
+      {tag: ['@northstar']},
+      async () => {
+        const {page, applicantQuestions} = ctx
+        await applicantQuestions.applyProgram(programName)
+
+        await test.step('Screenshot without errors', async () => {
+          await validateScreenshot(
+            page,
+            'id-north-star',
+            /* fullPage= */ true,
+            /* mobileScreenshot= */ true,
+          )
+        })
+
+        await test.step('Screenshot with errors', async () => {
+          await applicantQuestions.clickContinue()
+          await validateScreenshot(
+            page,
+            'id-errors-north-star',
+            /* fullPage= */ true,
+            /* mobileScreenshot= */ true,
+          )
+        })
+      },
+    )
+  })
+
+  async function setUpForSingleInput(programName: string) {
```

Comment:
suggestion (non-blocking):  This function name seems vague to me.  I also wonder if we need to mention that its for a single input since ID questions will always be a single input, no?  How about `setUpProgramWithIDQuestion`?